src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin/Microsoft.Commerce/UsageConnections/AddUsageConnection.cs

```diff
@@ -25,6 +25,7 @@ namespace Microsoft.AzureStack.Commands
     /// </summary>
     [Cmdlet(VerbsCommon.Add, Nouns.UsageConnection)]
     [OutputType(typeof(UsageConnectionModel))]
+    [Alias("Add-AzureRMUsageConnection")]
```

Comment:
Rm  PowerShell cmdlet names are not case sensitive, but we want to use the correct capitalization in the alias.